python/setup.py

```diff
@@ -105,42 +140,50 @@ def run(self):
         os.system('find . -name "*.cpython*.so" -type f -delete')
 
 
+class build_ext_no_debug(_build_ext):
```

Comment:
minor: I've always seen "release" used as the term for "no debug" (ie. it's either a release build or a debug build).

Reply:
I just noticed this is a hotfix for 21.08, so I don't mind if this change isn't made (I'm fine either way).  I'll approve.

Reply:
@ajschmidt8 is 21.08 and main the correct branches?

Reply:
yes, these are the correct branches. this needs to go into `branch-21.08`.